primerpedia.js

```diff
@@ -18,7 +18,7 @@
  *    and some templates are still displayed, so they need to be hidden via css
  */
 
-var apiUrl = "http://en.wikipedia.org/w/api.php?";
+var apiUrl = "https://en.wikipedia.org/w/api.php?";
```

Comment:
Could you leave this out for now? Github automatically redirects waldyrious.github.io to my website waldyrious.net, which is not setup for https yet, so this API query would raise a mixed content error.